can_tools/scrapers/official/AZ/AZ_TableauPull.py

```diff
@@ -0,0 +1,280 @@
+                "dataValues"
+            ][
+                -1
+            ]
+            intDat = htmdump["secondaryInfo"]["presModelMap"]["dataDictionary"][
+                "presModelHolder"
+            ]["genDataDictionaryPresModel"]["dataSegments"]["0"]["dataColumns"][0][
+                "dataValues"
+            ]
+            rlDat = htmdump["secondaryInfo"]["presModelMap"]["dataDictionary"][
+                "presModelHolder"
+            ]["genDataDictionaryPresModel"]["dataSegments"]["0"]["dataColumns"][1][
+                "dataValues"
+            ]
+            # First extract the datatype and indices:
+            for i in htmdump["secondaryInfo"]["presModelMap"]["vizData"][
+                "presModelHolder"
+            ]["genPresModelMapPresModel"]["presModelMap"]:
+                dtyp = htmdump["secondaryInfo"]["presModelMap"]["vizData"][
+                    "presModelHolder"
+                ]["genPresModelMapPresModel"]["presModelMap"][i]["presModelHolder"][
+                    "genVizDataPresModel"
+                ][
+                    "paneColumnsData"
+                ][
+                    "vizDataColumns"
+                ][
+                    1
+                ].get(
+                    "dataType"
+                )
+                indx = htmdump["secondaryInfo"]["presModelMap"]["vizData"][
+                    "presModelHolder"
+                ]["genPresModelMapPresModel"]["presModelMap"][i]["presModelHolder"][
+                    "genVizDataPresModel"
+                ][
+                    "paneColumnsData"
+                ][
+                    "paneColumnsList"
+                ][
+                    0
+                ][
+                    "vizPaneColumns"
+                ][
+                    1
+                ].get(
+                    "aliasIndices"
+                )[
+                    0
+                ]
+                if dtyp == "integer":
+                    valF.append([area, i, intDat[indx]])
+                elif dtyp == "real":
+                    valF.append([area, i, rlDat[indx]])
+            valF.append([area, "Last update", lsUpdt])
+            if valF:
+                val = pd.DataFrame(valF, columns=["location", "Name", "Value"])
+                val = pd.pivot_table(
+                    val,
+                    values="Value",
+                    index=["location"],
+                    columns="Name",
+                    aggfunc="first",
+                ).reset_index()
+                return val
+            else:
+                return None
+
+        # Let's start at the state-level
+        url = "https://tableau.azdhs.gov/views/COVID-19Summary/Overview2"
+        # Initialize main page: grab session ID key, sheet ID key, root directory string
+        r = requests.get(url, params=self.reqParams)
+
+        # Parse the output, return a json so we can build a bootstrap call
+        suppe = BeautifulSoup(r.text, "html.parser")
```

Comment:
Are we using anything special from `"html.parser"`? If not, I think we can use the `lxml` parser -- `lxml` is already listed as a dependency in the `requirements.txt`

Reply:
This is the html.parser that comes along with BS4 (no additional library install needed).  Did you want to move away from BS4 entirely in preference of lxml?  I can make these updates, but thought we already said BS4 was ok to add to the requirements list.

Reply:
I'm ok to have both as dependencies.

While they are quite similar in scope and we *could* use just one, I don't think it is necessary. Both are mature and easy to install so it shouldn't cause any problems for deployment

Reply:
Definitely okay with using BS4 and putting it in the requirements list -- This was more a comment that you can use different underlying parsers behind BS4. The [BS4 docs](https://beautiful-soup-4.readthedocs.io/en/latest/#installing-a-parser) claim that lxml is both flexible and fast but the cost is that it requires an external dependency so I just meant that since we already  have `lxml` as a dependency then do we want to use the `lxml` parser for BS4.

If speed isn't an issue then I wouldn't worry about it.

Reply:
Oh that makes sense! I'll see about using that instead, I thought you had meant as an alternative.